specs/am/architecture-management-shapes.ttl

```diff
@@ -30,11 +30,11 @@
   rdfs:label "OSLC Architecture Management (AM) Constraints" ;
   dcterms:description "All vocabulary URIs defined in the OSLC Architecture Management (AM) namespace."^^rdf:XMLLiteral ;
   dcterms:publisher <https://open-services.net/about/> ;
-  #dcterms:issued "2020-xx-xx"^^<http://www.w3.org/2001/XMLSchema#date> ;
+  dcterms:issued "2020-09-17"^^<http://www.w3.org/2001/XMLSchema#date> ;
   dcterms:license <http://www.apache.org/licenses/LICENSE-2.0> ;
   dcterms:source <https://github.com/oslc-op/oslc-specs/blob/master/specs/am/architecture-management-shapes.ttl> ;
   dcterms:isPartOf <https://oslc-op.github.io/oslc-specs/specs/am/architecture-management-spec.html> ;
```

Comment:
dcterms:source and dcterms:isPartOf need to be changed to point to their future locations on open-services. See https://github.com/oslc-op/oslc-specs/pull/407/files